src/plugins/instrument_fetch.js

```diff
@@ -158,42 +158,46 @@ class InstrumentFetch {
 
         return function (request, options = {}) {
             request = new Request(request, options);
-            const url = request.url;
             const opts = tracer.options();
 
-            if (!self._shouldTrace(tracer, url)) {
+            if (!self._shouldTrace(tracer, request.url)) {
                 return proxiedFetch.apply(null, arguments);
             }
 
             let span = tracer.startSpan('fetch');
             tracer.addActiveRootSpan(span);
 
+            const parsed = new URL(request.url);
             let tags = {
-                method : options && options.method ? options.method : 'GET',
-                url    : url,
-            };
-            if (url) {
-                tags.url_pathname = url.split('?')[0];
-            }
-
-            const fetchPayload = Object.assign({}, tags);
+                method : request.method,
+                url    : request.url,
 
+                // NOTE: purposefully excluding username:password URL properties.
```

Comment:
Should we do this? Felt wrong to include those possibly insecure properties (but then again this library can include cookies, so I'm not sure).

Reply:
I don't think we should include those either. They weren't there before and seem really strange to include. Although I guess if they were in the url, they must also be in the request.url which would not great. I guess the user in that case would have to blacklist those specific urls too.

Reply:
That is a good point, that regardless the user would have to exclude those URLs! I'll put in the username/password with a comment.

Reply:
I agree that since it's part of the URL we might as well include it here. For cookies we explicitly made it possible to exclude them using the `include_cookies` option for PII-reasons